src/main/java/spleetwaise/transaction/model/transaction/Date.java

```diff
@@ -15,9 +16,10 @@
 public class Date {
     public static final String MESSAGE_CONSTRAINTS = "Date should only in the format of DDMMYYYY";
```

Comment:
Should we change this to reflect that it should only be a valid date since iirc this is also used as an error message?